service/azureconfig/v1/arm_templates/security_groups_setup.json

```diff
@@ -41,6 +41,18 @@
     },
     "kubernetesIngressInsecurePort": {
       "type": "int"
+    },
+    "cadvisorPort": {
+        "type": "int"
+    },
```

Comment:
I would make those variables and not pass it from main. This makes templates more usable on their own. You can also set default values and omit the parameters if you think it's important to have them configurable.